src/pages/Beta/Swap/WatchList/index.tsx

```diff
@@ -0,0 +1,153 @@
+import React, { useContext, useRef, useState, useMemo } from 'react'
+import { Box, Button } from '@pangolindex/components'
+import { ChainId, Token, CHAINS } from '@pangolindex/sdk'
+import { Plus } from 'react-feather'
+import { ThemeContext } from 'styled-components'
+import { PNG } from 'src/constants/tokens'
+import { useActiveWeb3React } from 'src/hooks'
+import WatchlistRow from './WatchlistRow'
+import { WatchListRoot, GridContainer, Title, DesktopWatchList, MobileWatchList } from './styleds'
+import Scrollbars from 'react-custom-scrollbars'
+import CoinChart from './CoinChart'
+import CurrencyPopover from './CurrencyPopover'
+import { useOnClickOutside } from 'src/hooks/useOnClickOutside'
+import useToggle from 'src/hooks/useToggle'
+import { useSelectedCurrencyLists } from 'src/state/watchlists/hooks'
+import { useTranslation } from 'react-i18next'
+import { useAllTokens } from 'src/hooks/Tokens'
+import ShowMore from 'src/components/Beta/ShowMore'
+import { Hidden } from 'src/theme'
+
+type Props = {
+  isLimitOrders?: boolean
+}
+
+const WatchList: React.FC<Props> = ({ isLimitOrders }) => {
```

Comment:
what is the reason you have added Watchlist component back ? we have moved watchlist component to components repo